javascript/organizing-js/todo-project.md

```diff
@@ -23,9 +23,13 @@ Before diving into the code, take a minute to think about how you are going to w
    1. [Todoist](https://en.todoist.com/)
    2. [Things](https://culturedcode.com/things/)
    3. [any.do](https://www.any.do/)
-7. Use localStorage to save user's projects and todos between sessions.
-8. Since you are probably already using webpack, adding external libraries from npm is a cinch!  You might want to consider using the following useful library in your code:
+7. Since you are probably already using webpack, adding external libraries from npm is a cinch!  You might want to consider using the following useful library in your code:
    1. [date-fns](https://github.com/date-fns/date-fns) gives you a bunch of handy functions for formatting and manipulating dates and times.
+8. We haven't learned any techniques for actually storing our data anywhere, so when the user refreshes the page, all of their todos will disappear! If you want, you are capable of adding some persistence to this todo app using the Web Storage API.
```

Comment:
Should we keep the "if you want" or would it be better to make this a permanent part of the assignment? If we want this to remain optional, we should remove "if you want" and just say it's part of the assignment. If we want it to be an optional step, we should label it as "extra credit" or "Optional" (like in the library project).

Reply:
localStorage is supposed to be a part of the Todo List project requirement. I'll remove the "if you want". Thanks for pointing it out